Formula/at-spi2-core.rb

```diff
@@ -0,0 +1,36 @@
+class AtSpi2Core < Formula
+  desc "Protocol definitions and daemon for D-Bus at-spi"
+  homepage "https://www.freedesktop.org/wiki/Accessibility/AT-SPI2"
+  url "https://download.gnome.org/sources/at-spi2-core/2.36/at-spi2-core-2.36.1.tar.xz"
+  sha256 "97417b909dbbf000e7b21062a13b2f1fd52a336f5a53925bb26d27b65ace6c54"
+  license "LGPL-2.1-or-later"
+  revision 3
+
+  depends_on "gobject-introspection" => :build
+  depends_on "intltool" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python@3.9" => :build
+  depends_on "dbus"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "libx11"
+  depends_on "libxtst"
+  depends_on :linux
+  depends_on "xorgproto"
+
+  def install
+    ENV.refurbish_args
+
+    mkdir "build" do
+      system "meson", "--prefix=#{prefix}", "--libdir=#{lib}", ".."
+      system "ninja"
+      system "ninja", "install"
+    end
+  end
+
+  test do
+    system "#{libexec}/at-spi2-registryd", "-h"
```

Comment:
No better test possible? I'm a bit disappointed that most of the new Linux-only formulae being added have minimal testing.

Reply:
Yeah, I'm not too happy about the tests either.  I will try to make a better one here.

Reply:
I have a much better test for this now which compiles a small C program that includes one of the headers and links to the shared library, but I'm now stuck at the last step.  I want to use `assert_match` to look for `AT-SPI` in this output from running `#{testpath}/test`:
```
(process:18859): dbind-ERROR **: 09:02:38.116: AT-SPI: Couldn't connect to accessibility bus. Is at-spi-bus-launcher running?
```
but `assert_match "AT-SPI", shell_output("#{testpath}/test")` fails with 
```
An exception occurred within a child process:
  Minitest::Assertion: Expected: 0
  Actual: nil
```
I was following other examples I've seen of test like this in homebrew-core, but there must be some simple mistake I'm making.  I can just push the change if that would make it easier to troubleshoot.

Reply:
It probably writes an error to stderr instead of stdout which `shell_output` captures. `&>` is your friend

Reply:
That was it!  Thank you so much.  I've pushed the new and improved test which I think should be in line with what we usually expect in homebrew-core.